app/routes/task_routes.py

```diff
@@ -0,0 +1,116 @@
+from app import db
+from app.models.task import Task
+from .route_helpers import *
```

Comment:
We should try to avoid `import *` since this could end up bringing in an unexpected function/variable that collides with something in our own code without us knowing it. With `import *`, any time we make a change to `route_helpers`, there's a chance we could unintentionally break code elsewhere in our project.